src/pages/landing/ebooks/components/_main-head.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```

Comment:
If we need this `Head` logic inside other components let's move it to `src/components/containers` but if we only need it inside landing/ebooks pages let's keep it here.
Rest LGTM.